requirements.txt

```diff
@@ -1,7 +1,7 @@
 cycler
 event-model>=1.5.0,!=1.8.0
 historydict
-jsonschema<3
+jsonschema
```

Comment:
I believe that, with this change, `jsonschema` is longer a direct dependency of bluesky (i.e. we don't import it anymore) and can therefore be removed from this list. It will come in through our `event-model` dependency and we can manage the `jsonschema` pinning there, which is easier than keeping both libraries in sync.

Reply:
I did not notice that.